### DISCUSSION TITLE
🛋️ Local co-op: list of supported/unsupported games on xCloud

### DISCUSSION THREAD

**Discussion by redphx** [redphx/better-xcloud#275] [General]
> [!WARNING]
> - This list only applies to xCloud. Remote Play should supports all local co-op games, except for some (ex: Minecraft).
> - The local co-op feature won't work with games that ***require additional profile***. Only Xbox can add support for these games. Don't ask me for this feature.

![image](https://github.com/user-attachments/assets/0c283f99-797c-4a0b-b12b-d25cea508cfc)
![image](https://github.com/user-attachments/assets/0e729ab7-9bf5-4c42-88b8-d5f6cb192755)

- This feature is only available when using [Better xCloud](https://better-xcloud.github.io/local-co-op/).
- You can also use it with the Mouse & Keyboard feature.

> [!NOTE]
> If you tested a game that wasn't in this list, please comment below. Thanks.

### Supported games  
- [As Dusk Falls](https://www.xbox.com/en-US/play/games/as-dusk-falls/9NR7XDNVP5SW)
- [Banjo-Tooie](https://www.xbox.com/en-US/play/games/banjo-tooie/BX27S00SKW1V)
- [Ben 10: Power Trip](https://www.xbox.com/en-US/play/games/ben-10-power-trip/9NJWFF2KHL6H)
- [BlazBlue: Cross Tag Battle Special Edition](https://www.xbox.com/play/games/blazblue-cross-tag-battle-special-edition/9NFWDRBZXDL9)
- Bluey
- [Broforce](https://www.xbox.com/play/games/broforce/9P03ZQMBMV65)
- [Chicory: A Colorful Tale](https://www.xbox.com/play/games/chicory-a-colorful-tale/9PFGQGC0XWLV)
- Crimson Skies
- 😈 [DOOM (1993)](https://www.xbox.com/en-US/play/games/doom-1993/9PLZPHBNHTMF)
- 😈 [DOOM II (Classic)](https://www.xbox.com/en-US/play/games/doom-ii-classic/9PLT62LRF9V7)
- ⚽️ EA SPORTS FC/FIFA
- [Escape Academy](https://www.xbox.com/play/games/escape-academy/9P3NT9H51RMR)
- Enter the Gungeon
- [Figment 2: Creed Valley](https://www.xbox.com/en-US/play/games/figment-2-creed-valley/9NZWW8K1KX6S)
- [Fuzion Frenzy](https://www.xbox.com/play/games/fuzion-frenzy/C2P985H1H42H)
- [Gang Beasts](https://www.xbox.com/play/games/gang-beasts/BPQZT43FWD49)
- ⚙️ [Gears of War 2](https://www.xbox.com/play/games/gears-of-war-2/C1SDBNRFXT1D)
- ⚙️ [Gears of War 3](https://www.xbox.com/play/games/gears-of-war-3/BPKDQSSFQ9WV)
- ⚙️ [Gears of War: Judgment](https://www.xbox.com/play/games/gears-of-war-judgment/BSHMMGRP84N4)
- [Goat Simulator](https://www.xbox.com/play/games/goat-simulator/BV0NM309K1TL)
- [GoldenEye 007](https://www.xbox.com/play/games/goldeneye-007/9N6639H7VGH4)
- 👊 [Injustice 2](https://www.xbox.com/play/games/injustice-2/BPKVH4C4XV4N) (press B when selecting profile for Player 2 -> "Continue without saving")
- [JoJo's Bizarre Adventure: All-Star Battle R](https://www.xbox.com/en-US/play/games/jojo's-bizarre-adventure-all-star-battle-r/9P4811KBD5R9)
- Joy Ride Turbo
- [Killer Instinct](https://www.xbox.com/play/games/killer-instinct-anniversary-edition/9N2HB4MWG7HM)
- Kingdom Two Crowns
- [LEGO® Star Wars™: The Skywalker Saga](https://www.xbox.com/play/games/lego-star-wars-the-skywalker-saga/BRMKDCZT0C4L)
- 🏈 [Madden NFL](https://www.xbox.com/play/search?q=madden)
- ⚾️ [MLB The Show](https://www.xbox.com/play/search?q=mlb)
- 🩸 [Mortal Kombat 11](https://www.xbox.com/play/games/mortal-kombat-11/BTC0L0BW6LWC) (press B when selecting profile for Player 2 -> "Kontinue without saving")
- Mortal Kombat 1
- 🏀 [NBA 2K24](https://www.xbox.com/play/games/nba-2k24-for-xbox-series-x%7Cs/9NT7FNZ0WJM1)
- NHL
- Nickelodeon All-Star Brawl 2  
- [Party Animals](https://www.xbox.com/play/games/party-animals/9MZ08RDQ4DM1)
- [PAW Patrol Mighty Pups Save Adventure Bay](https://www.xbox.com/en-US/play/games/paw-patrol-mighty-pups-save-adventure-bay/9N3TF03KNTBD)
- [Pikuniku](https://www.xbox.com/en-US/play/games/pikuniku/9N8WRDC25K6J)
- [Rush: A Disney Pixar Adventure](https://www.xbox.com/en-US/play/games/rush-a-disneypixar-adventure/9P3PL76N0KWZ)
- [Serious Sam Collection](https://www.xbox.com/play/games/serious-sam-collection/C12T2QLN7H4V)
- [Soccer Story](https://www.xbox.com/play/games/soccer-story/9PJ1045MHZJ0)
- [Spiritfarer](https://www.xbox.com/play/games/spiritfarer-farewell-edition/9NG5L58FD3X5)
- [Stardew Valley](https://www.xbox.com/play/games/stardew-valley/C3D891Z6TNQM)
- [STAR WARS™ Battlefront™ II](https://www.xbox.com/play/games/star-wars-battlefront-ii/C0GWTPD0S8S1)
- [SpiderHeck](https://www.xbox.com/play/games/spiderheck/9N0TRF57SMQH)
- 🐢 [Teenage Mutant Ninja Turtles](https://www.xbox.com/play/games/teenage-mutant-ninja-turtles-shredder's-revenge/9NS3673HVH41)
- [Totally Reliable Delivery Service](https://www.xbox.com/play/games/totally-reliable-delivery-service/9NZG72SH3H4W)
- UFC 5
- [Vampire Survivors](https://www.xbox.com/play/games/vampire-survivors/9PD5BM2Z8C4L)
- WWE 2K24

### Unsupported games  
- [Arcade Paradise](https://www.xbox.com/en-US/play/games/arcade-paradise/9N2FQCGWH32D)
- [ARK](https://www.xbox.com/play/games/ark-ultimate-survivor-edition/9N5JRWWGMS1R)
- Baldur's Gate 3
- [Battletoads](https://www.xbox.com/play/games/battletoads/9N7GCF5SGCXC)
- [Bluey: The Videogame](https://www.xbox.com/en-US/play/games/bluey-the-videogame/9N2J0FGV30DX)
- Call of Duty games
- Borderlands 3
- Cult of the Lamb
- [DiRT 5](https://www.xbox.com/play/games/dirt-5/9PJGM0T0827V)
- Fornite
- [It Takes Two](https://www.xbox.com/play/games/it-takes-two---digital-version/9NXVC0482QS5)
- Halo: all games
- [HOT WHEELS UNLEASHED™](https://www.xbox.com/en-US/play/games/hot-wheels-unleashed---game-of-the-year-edition---/9NGZ7PN3W8DX)
- [HOT WHEELS UNLEASHED™ 2 - Turbocharged](https://www.xbox.com/en-US/play/games/hot-wheels-unleashed-2---turbocharged/9MWDKL3SD6D9)
- [Human: Fall Flat](https://www.xbox.com/play/games/human-fall-flat/BSMZH25V6V46)
- [Goat Simulator 3](https://www.xbox.com/play/games/goat-simulator-3/9NMB4T25W817)
- [Guilty Gear -Strive-](https://www.xbox.com/play/games/guilty-gear--strive-/9MZDL6L34KF2)
- LEGO 2K Drive
- [Minecraft Dungeon](https://www.xbox.com/play/games/minecraft-dungeons/9N8NJ74FZTG9)
- [Overcooked 2](https://www.xbox.com/play/games/overcooked-2/BVJLKDG2TX8H)
- [PAW Patrol World](https://www.xbox.com/en-US/play/games/paw-patrol-world/9N2VDS259QP1)
- [Plants vs. Zombies Garden Warfare](https://www.xbox.com/play/games/plants-vs.-zombies-garden-warfare/BTJ0T8C04ZBV)
- [Plants vs. Zombies Garden Warfare 2](https://www.xbox.com/play/games/plants-vs.-zombies-garden-warfare-2/BNRH7BRC1D02)
- [Plants vs. Zombies: Battle for Neighborville](https://www.xbox.com/play/games/plants-vs.-zombies-battle-for-neighborville/C4HZC7LJG6PX)
- [PlateUp!](https://www.xbox.com/play/games/plateup/9P2HHT3VS7MP)
- [Peggle 2](https://www.xbox.com/play/games/peggle-2/BNCZHKWRZ7BR)
- [Quake](https://www.xbox.com/play/games/quake/9P1Z43KRNQD4)
- [Quake II](https://www.xbox.com/play/games/quake-ii/9P7L9H478GGV)
- [Spacelines from the Far Out](https://www.xbox.com/play/games/spacelines-from-the-far-out/9N23WV1HGLTQ)
- [SpongeBob SquarePants: Battle for Bikini Bottom - Rehydrated](https://www.xbox.com/en-US/play/games/spongebob-squarepants-battle-for-bikini-bottom---r/C35S5BRNWL70)
- [Unravel Two](https://www.xbox.com/play/games/unravel-two/C4VKLMG1HLZW)

### Not tested
- [Cassette Beasts](https://www.xbox.com/en-US/play/games/cassette-beasts/9N82SNQTZVMQ)
- [Cricket 22](https://www.xbox.com/en-US/play/games/cricket-22/9NWNX54ZMT1K)
- [Disneyland Adventurers](https://www.xbox.com/en-US/play/games/disneyland-adventures/9N6Z8DQXSQWH)
- [Fable II](https://www.xbox.com/en-US/play/games/fable-ii/C2WKJJ9F5936)
- [Fable III](https://www.xbox.com/en-US/play/games/fable-iii/BR46KM4D5B9L)
- Gears: some games
- [Ghostlore](https://www.xbox.com/en-US/play/games/ghostlore/9P2Z4ZP862VP)
- [RESEARCH and DESTROY](https://www.xbox.com/play/games/research-and-destroy/9NNZJ389GSW2)
- [Super Mega Baseball 3](https://www.xbox.com/play/games/super-mega-baseball-3/9NP6V9LFVJ87)
- [Super Mega Baseball 4](https://www.xbox.com/play/games/super-mega-baseball-4/9N93V5M3TG4N)

**Comment by redphx** [comment]
Supported: [Fuzion Frenzy](https://www.xbox.com/en-US/play/games/fuzion-frenzy/C2P985H1H42H)

**Reply by santygiral20305** [reply]
Gears 2 AND gears 3 working

**Reply by playstationxones** [reply]
Ben 10: Power Trip Working

**Reply by Ghitusss** [reply]
The dlc of brotato is supported

**Reply by weslleyfrancisco201-sys** [reply]
When will there be Cuphead?

**Reply by Jefflou45** [reply]
College football works

**Reply by Stef-Iaco-Dev** [reply]
Excuse me, when will there be LEGO Batman 3 Beyond Gotham? Are you working in that?

**Comment by waiterss1** [comment]
halo 4 não suportado

**Comment by redphx** [comment]
Unsupported:
- All Halo games
- ~~All Gears games~~
- Fortnite

**Comment by redphx** [comment]
Supported:
- [BlazBlue: Cross Tag Battle Special Edition](https://www.xbox.com/en-US/play/games/blazblue-cross-tag-battle-special-edition/9NFWDRBZXDL9)
- [LEGO® Star Wars™: The Skywalker Saga](https://www.xbox.com/en-US/play/games/lego-star-wars-the-skywalker-saga/BRMKDCZT0C4L)

Unsupported:
- [Guilty Gear -Strive-](https://www.xbox.com/en-US/play/games/guilty-gear--strive-/9MZDL6L34KF2)

**Comment by miguel13232** [comment]
Unsupported

 ARK survival evolved

**Comment by Emerrsondsr** [comment]
Gang Beasts Suportado no modo local

**Reply by redphx** [reply]
Thanks. I tested it before but forgot to add it to the list.

**Comment by SamuRey1** [comment]
the local mode of the Killer Instinct videogame works
![Killer INstinct](https://github.com/redphx/better-xcloud/assets/162636478/5e173157-dc62-40fe-ac60-9a6ab9d10420)

**Reply by redphx** [reply]
Added. Thanks.

**Comment by santygiral20305** [comment]
Gears 2 3 AND judment working
AND Star wars battlefront

**Reply by redphx** [reply]
Thanks. Not sure why Gears games didn't work for me before.

**Reply by santygiral20305** [reply]
It's xbox 360 you can use guest user option

**Comment by MarbleRabbit** [comment]
Jojo Bizarre Adventure:All-Star Battle R working

**Reply by redphx** [reply]
Added. Thanks!

**Comment by redphx** [comment]
Unsupported:
- DiRT 5
- HOT WHEELS UNLEASHED™
- PAW Patrol World

**Reply by Prodreus** [reply]
How did you manage to play? I am unable

**Comment by redphx** [comment]
Supported:
- DOOM (1993)
- DOOM II (Classic)
- Figment 2
- Pikuniku

Unsupported:
- Arcade Paradise
- Bluey

Not tested:
- As Dusk Falls
- Ben 10
- Cassette Beasts
- Cricket 22
- Disneyland Adventurers
- Fable II
- Fable III
- Ghostlore
- PAW PAtrol Mighty Pups
- Plants vs. Zombies Garden Warfare
- Plants vs. Zombies Garden Warfare 2
- Plants vs. Zombies: Battle for Neighborville™
- RESEARCH and DESTROY
- Soccer Story
- Super Mega Baseball 3
- Super Mega Baseball 4

**Comment by gabrielviniciusnunes** [comment]
Supported:
NBA 2K24

**Comment by redphx** [comment]
Unsupported:  
SpongeBob SquarePants: Battle for Bikini Bottom - Rehydrated

**Comment by santygiral20305** [comment]
Unsupported: All pvz

**Comment by emitasxs** [comment]
As dusk fall supported

**Comment by Tak-attack** [comment]
Supported: Banjo-Tooie

**Comment by Rony9u3** [comment]
Supported:PAW PAtrol Mighty Pups,fable II and fable III

**Reply by redphx** [reply]
I tested Paw Patrol before and it didn't work. Did they update the game?

**Reply by Soufflesec** [reply]
I tested. I confirm that Paw Patrol Mighty Pups is supported. You need to launch the first mission, then after some seconds (short tutorial) the option appears in-game ("press --- to join Player") during the mission.

Indeed, you don't rule the coop on main/title menu before missions. The second player must join the party.

You can watch this video for the first mission : https://youtu.be/J2kWl7rmus4

**Reply by redphx** [reply]
Oh the game I tested was PAW Patrol World

**Comment by Soufflesec** [comment]
Hello.

I tested a few games that are supposed to be supported. I confirm that Goatsimulator 1, Injustice 2, Lego Star Wars Skywalker Saga and TMNT work in local co-op. 

But Gears 2 and 3 do not work for me contrary to what is said. I tried Gears 5, same result.

Thanks.

**Reply by santygiral20305** [reply]
Bro gears work, gears 4, 5 AND ultimate edition They don't work (at the moment) because If I remember correctly they never asked me for a profile on Xbox One, but gears 2, 3 AND judment works

**Reply by Soufflesec** [reply]
Thank you for the answer. I'll try a third time. I will take a screenshot if the second controller is not recognized.

**Comment by santygiral20305** [comment]
I think xbox cloud Is working to add More Profiles option in the service (To improve the split screen option)

**Reply by redphx** [reply]
Where do you get this info? Or you're just guessing?

**Reply by santygiral20305** [reply]
I'm guessing obviously

**Reply by byomanisme** [reply]
![Screenshot_2024-06-26-18-00-22-03_7cbc8fc42dc24310c089978df634fb69](https://github.com/redphx/better-xcloud/assets/173902519/ce15555a-c002-4663-ba08-83cbb41ebde2)

If only I could add another account on this page, I might be able to play It Take Two Game in Local Co-op.

**Comment by gsh125** [comment]
Soccer Story support
![soccer-story-1711192992871](https://github.com/redphx/better-xcloud/assets/162846577/353312b0-2019-4e39-a4ea-48cf92a05654)

**Reply by redphx** [reply]
Added

**Comment by santygiral20305** [comment]
![IMG_20240324_145115_201](https://github.com/redphx/better-xcloud/assets/101019541/0b63d74f-80fc-4f39-9a7d-4c67ee9661b6)
My proof that gears works on xcloud

**Reply by santygiral20305** [reply]
Soccer story working

**Reply by Soufflesec** [reply]
Soccer Story Working seems to have already been added.

Regarding Gears, which browser do you use (Chrome for PC or Kiwi for Android, even though I don't think it matters)? Another question: do you log into your Live account when you launch the game? I didn't have time to try again to take a screenshot showing that the second controller is not recognized in my case (Kiwi on Android, Live account connected when launching the game). I'll try again soon.

**Comment by Soufflesec** [comment]
Has anyone else success to activate multi on Gears 2/3/Judgment ?

**Reply by santygiral20305** [reply]
Yes I already do it see my image below in these posts

**Reply by Soufflesec** [reply]
Unfortunately few people participate. I wanted to see if anyone other than you had managed to activate multiplayer to compare (I asked you questions below but you haven't reply yet).

![Screenshot_20240331_183124_Kiwi Browser](https://github.com/redphx/better-xcloud/assets/163769033/0d4fb354-ba2f-45d0-b8ec-ad979b776370)
![Screenshot_20240331_183140_Kiwi Browser](https://github.com/redphx/better-xcloud/assets/163769033/bf2f7184-ed51-4100-a9a4-7196b14dfc19)
![Screenshot_20240331_183148_Kiwi Browser](https://github.com/redphx/better-xcloud/assets/163769033/619f69ff-4b37-4c49-8e22-2b81427719a0)

As you can see, my second controller isn't recognized in Gears of War games.

**Reply by santygiral20305** [reply]
Idk

**Comment by DeivisomFrigeri** [comment]
rush a disney pixar adventure supported

**Comment by jeffgedler** [comment]
Nice work!!   I'm just wondering if the rest of the gears will be playable multiplayer and dead island 2 would be interesting the possibility of multiplayer

**Comment by santygiral20305** [comment]
https://github.com/redphx/better-xcloud/assets/101019541/e77d9e66-8677-4bb7-ab53-bbbbd7582c53

Another prove gears works

**Reply by Soufflesec** [reply]
To be proof, you must film from the beginning: xcloud menu, select the game, and finally arrive at this local coop validation screen. By filming the entire screen it would be even better.

**Comment by Vitover** [comment]
My games isnt working, party animals and goat simulator , but they are on supported, someone know what's happening?

**Reply by santygiral20305** [reply]
I don't understand what you're saying, be clearer.

**Reply by Vitover** [reply]
The coop on this games dont work

**Reply by Soufflesec** [reply]
Hello.

What controllers do you use? Do each work on single player games on xcloud?

I tried Goat Simulator a few weeks ago : it worked. I'll try again in a few hours because I don't remember if there was a particular procedure, then I will come back to give you an answer.

I'll try Party Animals too.

**Reply by redphx** [reply]
It's working for me. Remember, it only supports Goat Simulator 1, not 3.  

<img width="738" alt="image" src="https://github.com/redphx/better-xcloud/assets/96280/3285c794-f6a9-48a5-b747-269410b16205">

**Reply by DeivisomFrigeri** [reply]
Hey guys! Are you able to play Hotwheels 2 with 2 controllers? Here for me, only one is working... Is there any trick to make the local coop in this game work?

**Comment by Vitover** [comment]
My party animals , the goat simulator works normally but party animals dont work, we use different controllers my friend use a ps2 controller and me a xbox 360 ,both generic

**Comment by leuuu1997** [comment]
Will you  guys update more games for local co-op? Specifically, the games that are already available on Game Pass?

**Reply by Soufflesec** [reply]
I don't think that he can add games manualy.

To be played on local coop, games must meet certain technical characteristics.
Redphx opened up the possibility for us to play these games. For other games, I think it's a different problem.

Maybe Redphx will confirm.

**Reply by leuuu1997** [reply]
oh, ok, got it... so the game needs to be able to works by default in local co-op. thanks

**Reply by Soufflesec** [reply]
The game must be local coop of course.

But as santygiral20305 says above, there may be local coop games that require a profile, others not. Microsoft should enable the feature on all games in the future (maybe this year?).

In the meantime, Redphx already allows us to unlock this function on certain games.

**Reply by leuuu1997** [reply]
Yes, absolutely, it's fantastic; many thanks to their team.

**Comment by LittleRusty** [comment]
NHL 24 Worked

**Comment by spesknight** [comment]
Lego 2k Drive doesn't work, sadly

**Comment by Henry024545** [comment]
should'nt hot wheels unleashed 2 be working?

**Comment by jirigrigar** [comment]
Nickelodeon All-Star Brawl 2 working

**Comment by redphx** [comment]
Worked:  
- Crimson Skies
- Joy Ride Turbo

**Comment by nuarho** [comment]
not supported
F1 23 
PAW Patrol : Grandprix

**Comment by skfjah** [comment]
hello is eafc24 still supported? how can i do it?

**Reply by lebggxbox** [reply]
Yes you can. Add me in discord or snapchat i can explain. Snap : Dildarmayot2020 Discord : didililo

**Reply by mohammedsaqib1** [reply]
Please tell how

**Reply by mohammedsaqib1** [reply]
I m thinking of xbox cloud game pass and play fc24 with two local controllers. How can I do it?

**Comment by Aphador** [comment]
I played Bluey The Videogame and Disneyland Adventures with my son and both are working 2 player.

**Comment by ThallesCansi** [comment]
Black Ops 6 still support? Anyone tested?

**Reply by redphx** [reply]
Not supported

**Reply by pedrodevvnk** [reply]
Does F1 23 work?

**Comment by PedroLeal16** [comment]
Gears of war: Ultimate Edition works co op local split screen?

**Reply by santygiral20305** [reply]
No, it doesn't work

**Comment by Dougrasss1** [comment]
Working:

Mad Streets
All You Need is Help (press Y on the first controller to enable the secound player, but the secound player will have to move with the right analog stick on his gamepad)

**Comment by SUPR3M01** [comment]
Is Mortal Kombat 1 working?

**Comment by SUPR3M01** [comment]
mortal kombat 1 is working I just tested it
 ![Xbox Cloud Gaming - Mortal Kombat™ 1 _ Xbox Cloud Gaming (Beta) em  Xbox com 01_12_2024 11_56_15](https://github.com/user-attachments/assets/5f6808b5-1734-411c-a45c-09b584f12df8)

**Comment by nuarho** [comment]
Unsupported 

Gears of war 4
Gears 5
F1 23
Hot wheels Turbo Charged 2

**Comment by Bgtlep** [comment]
Hello, since Baldur game 3 can now be played on xcloud when you own it, has anyone tested if it can be played coop on xcloud as well?

**Reply by Bgtlep** [reply]
Xcloud on game pass ultimate I mean. I have game pass ultimate but on a Xbox serie S which does not support local coop split screen. I would like to play bg3 local coop on the cloud. Is it possible ?

**Reply by LLxD** [reply]
BG3 coop doesn't seem to work, unfortunately :cry:

**Comment by Waph1** [comment]
Supported Cassette Beasts

**Comment by melwynfp** [comment]
Is WWE 2K24 supported?

**Reply by melwynfp** [reply]
Bought it and tried it myself. Local co-op works :)

**Comment by mutatisIV** [comment]
supported UFC 5

**Reply by Benz2124** [reply]
I use xcloud on my computer. How can I play UFC 5 in local coop?

**Comment by AlexandreBMelo** [comment]
supported Kingdom Two Crowns
![image](https://github.com/user-attachments/assets/3df93e88-f5c6-43bf-9b44-4e84f8ea72f9)

**Comment by sayaslan** [comment]
Supported enter the gungeon

**Comment by Mtbjosh** [comment]
Borderlands 3 doesn't work

**Comment by ryodan95** [comment]
Serius sam collection working 2 players same profile, with split screen.
Cult of the lamb not work.

**Comment by Superbrothers5576** [comment]
Is Wobbly life working?

**Comment by MatheusBS2011** [comment]
Works with Lego Harry Potter?
In steam your own game

**Comment by ArithmeticLogicUnit** [comment]
Ultimate Chicken Horse can work with shared controller and 2 players same profile

**Comment by jirigrigar** [comment]
Unsupported:

- [Tony Hawk's Pro Skater 3+4](https://www.xbox.com/en-US/play/games/tony-hawk's-pro-skater-3-4---cross-gen-edition/9N926481VT1Z)

**Comment by Wellih** [comment]
Supported
Rubber bandits
<img width="1280" height="720" alt="rubber-bandits-1754081089182" src="https://github.com/user-attachments/assets/21b9f015-1500-4cfb-8bce-e6a2de43f83c" />

**Comment by XxxCoutinho** [comment]
Teenage Mutant Ninja Turtles mutants unleashed working 15/08/25

**Comment by EVAxTEST** [comment]
<img width="1280" height="609" alt="image" src="https://github.com/user-attachments/assets/8ddb4adc-d71b-4bf8-b333-a0c6d514e23f" />

Mighty Morphin Power Rangers: Rita's Rewind Working 10/09/25

**Comment by ECDLA** [comment]
supported local coop

Disney Speedstorm
<img width="1599" height="899" alt="image" src="https://github.com/user-attachments/assets/29f74593-c993-4017-a70f-00f5b5d26db3" />

**Reply by DanielSenaDev** [reply]
Yes, it works on Disney Speedstorm! You need to reach "Level 4" to play locally.

**Comment by vinicius-bastos** [comment]
F1 works, I`ve tested it and it does not, but I'm not sure

**Comment by alvarosamuel111-crypto** [comment]
Human: Fall Flat compatible

**Reply by Jefflou45** [reply]
Not working

**Comment by bernny2995** [comment]
**Supported:**

MONOPOLY 2024
ODD BALLERS
RABBIDS PARTY OF LEGENDS
RAYMAN LEGENDS


<img width="1365" height="763" alt="MONOPOLY 2024" src="https://github.com/user-attachments/assets/22bd70f5-dfcb-4ddf-bd7d-9645afea94c3" />
<img width="1365" height="755" alt="ODD BALLERS" src="https://github.com/user-attachments/assets/f69c0f35-e70c-4ea7-b7fe-8037381ae301" />
<img width="1360" height="767" alt="RABBIDS PARTY" src="https://github.com/user-attachments/assets/c16efcaa-9703-497c-b92f-8526f54a93d0" />
<img width="1364" height="767" alt="RAYMAN LEGENDS" src="https://github.com/user-attachments/assets/030a46cb-b182-4866-82bb-af5cf68cac5f" />

**Comment by bernny2995** [comment]
**Supported:**

MONOPOLY MADNESS 
STARLINK BATTLE FOR ATLAS
TRACKMANIA TURBO

<img width="1364" height="767" alt="MONOPOLY MADNESS" src="https://github.com/user-attachments/assets/ef26c4d0-2921-4885-b653-a72350265e3b" />
<img width="1365" height="767" alt="STARLINK" src="https://github.com/user-attachments/assets/4a99660f-c66d-4783-a2f7-2b5928e60052" />
<img width="1364" height="767" alt="TRACKMANIA TURBO" src="https://github.com/user-attachments/assets/5e4cdd64-d223-4f82-938e-79cdb5ec325f" />

**Reply by Jefflou45** [reply]
College football working

**Comment by thesnoc** [comment]
Tested For The King 2 on Android (Ayn Thor) with 4 controllers connected.
The game treated all controllers as one. Any controller could control any player. Not ideal but it works.

**Comment by MaxKaySam** [comment]
<img width="1440" height="900" alt="Captura de tela 2025-12-17 145112" src="https://github.com/user-attachments/assets/c0e4eae7-4ea4-4b9b-a2a0-3d5c75845d8e" />
Ultimate Sheep Racoon works

**Comment by 7mk8ybkymh-alt** [comment]
Working:
Ultimate Chicken Horse
Gang Beasts

**Comment by smromain** [comment]
Uno is supported

**Comment by edgar90011-maker** [comment]
What about it takes 2

**Comment by iagoalbuquerque97-ship-it** [comment]
I'm thinking about getting Game Pass to play on xCloud. I mainly want it to play It Takes Two. Is it possible to play it in co-op yet?

**Comment by Robsoon93** [comment]
WHAT about UFC5, anyone know how to play on two controlers?

**Reply by allenlawrence** [reply]
just go to fight now -> select a match -> select sides -> select players -> go!

**Comment by Biowar01** [comment]
I have a question, sorry. Can I play Fable 2 coop with parsec? My friend would play with mouse and keyboard while I would play with controller... Is that possible?

**Reply by 0DeadInside0** [reply]
fable 2 not supported mouse and keyboard only gamepad and with parsec u can use 2 gamepad i played ufc5 with my friend through parsec it supports playing with two gamepad like this

**Comment by Sauderzin** [comment]
Title: Remote Play black screen (GameStreamError: Failed to connect to game stream)
Description:
I'm using Better xCloud v6.7.12 (official Tampermonkey script).
Remote Play works perfectly without Better xCloud, but with Better xCloud it always shows the Xbox startup animation, then a black screen with no audio.
Console error:
WebRtcStream: Connect.
GameStreamError: Failed to connect to game stream
chrome://webrtc-internals shows the connection stuck at:
ICE connection state: checking
Connection state: connecting
DTLS state: new
The ICE connection never reaches connected or completed, and all DataChannels remain in connecting.
Environment:
Windows 11
Chrome and Edge (same behavior)
Better xCloud 6.7.12 (official script)
Xbox connected via Wi-Fi
NAT Open
Remote Play enabled
Tests performed:
Chrome and Edge
Disabled hardware acceleration
Disabled IPv6
Changed DNS (Cloudflare)
Reset Better xCloud settings
Chrome flags all on Default
Different Wi-Fi networks
Mobile 5G
Results:
✅ Official Remote Play works on all tested networks.
✅ Better xCloud works on 5G.
❌ Better xCloud fails on Wi-Fi networks (black screen after Xbox animation).
This looks like a WebRTC/ICE negotiation issue specific to Better xCloud, since the official Remote Play works correctly on the same device and network.

**Reply by vraafz** [reply]
<img width="4080" height="3060" alt="20260824_215335" src="https://github.com/user-attachments/assets/172fdf51-c581-4f6e-822b-e0030a39d4dc" />

🇧🇷 Português:
Lego Marvel super Heroes 2 funcionando corretamente o multijogador local. Creio que toda franquia lego suporte.

🇺🇸 English:
LEGO Marvel Super Heroes 2 — local multiplayer is working correctly. I believe the entire LEGO franchise supports local multiplayer.

🇪🇸 Español:
LEGO Marvel Super Heroes 2 — el modo multijugador local funciona correctamente. Creo que toda la franquicia LEGO admite multijugador local.
